core/kazoo_services/src/kz_services.erl

```diff
@@ -592,7 +592,11 @@ select_bookkeeper(BillingId) ->
     ResellerId = get_reseller_id(BillingId),
     {'ok', MasterAccountId} = kapps_util:get_master_account_id(),
     case ResellerId =/= MasterAccountId of
-        'true' -> 'kz_bookkeeper_local';
+        'true' ->
+            case ?KZ_RESELLER_BOOKKEEPER_LOOKUP of
+                'true' -> ?KZ_ACCOUNT_BOOKKEEPER(BillingId);
```

Comment:
@k-anderson agreed, will do it asap. Was thinking about BillingId as an AccountId of another child of the same reseller which means the same bookkeeper at the end of the day...
And yep, setting BillingId as an AccountId from another sub/resellers thread will make the game much more complex :)))
Thanks for an explanation, Karl!